src/cliConfig.ts

```diff
@@ -11,17 +12,22 @@ type CliConfig = {
 
 export default (argv: Array<string>): CliConfig => {
 	const program = new commander.Command();
+	let packageFile = "";
 
 	program
 		.version(`${version}`)
+		.arguments("<packageFile>")
```

Comment:
I believe we could change the notation to the following because the `packageFile` is in fact an optional parameter -- what do you think?

```suggestion
		.arguments("[packageFile]")
```

I've briefly looked at https://github.com/tj/commander.js#specify-the-argument-syntax 

> You use `.arguments` to specify the arguments for the top-level command, and for subcommands they are included in the .command call. Angled brackets (e.g. <required>) indicate required input. Square brackets (e.g. [optional]) indicate optional input.